### ISSUE TITLE
Adding Hex Colour Code for Calendars and Tasks

### ISSUE THREAD

**Issue by lasscap** [etesync/android#91]
I would love to have the option not just to pick a colour by sliding through the colour spectrum but also to use a hex code for the colours. Then I can (more easily) set the same colour for a related tasks-list and a calendar or use a colour I created on a more advanced colour spectrum.

**Comment by tasn** [comment]
Yeah, I agree it would be useful, and I also dislike color pickers who don't have the hex code.
We are however using a 3rd party library for that, would need to check if they support it, or evaluate another one if they don't.

**Comment by julianfairfax** [comment]
hi, have you had time to check if the 3rd party library supports it, or evaluate if another one does if they don't?

**Comment by tasn** [comment]
Either I have and forgot to update it, or I haven't and forgot about it. Either way though, I don't have time to implement it atm.
Happy to accept contributions for it though!
